packages/cli/generation/ir-generator/src/converters/services/convertHttpResponse.ts

```diff
@@ -42,6 +42,28 @@ export function convertHttpResponse({
     return undefined;
 }
 
+function convertJsonResponse(
+    response: RawSchemas.HttpResponseSchema | string,
+    docs: string | undefined,
+    file: FernFileContext,
+    typeResolver: TypeResolver
+): HttpResponse {
+    const responseBodyType = file.parseTypeReference(response);
+    const responseProperty = typeof response !== "string" ? response.property : undefined;
+    if (
+        responseProperty !== undefined &&
```

Comment:
nwe use `!= null` checks everywhere in case the value is ever null at runtime (!= null handles checking null and undefined)